Makefile

```diff
@@ -0,0 +1,14 @@
+GENERATED=	hcert_spec.html \
+		hcert_spec.docx
+
+
+all: $(GENERATED)
```

Comment:
Please try building and check results. I still have not fixed automatic generation and binary release attachment, so that needs to be performed manually for this release.